diagnostic.md

```diff
@@ -3,36 +3,38 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Adding routes to SPAs.
+Adding views to SPAs.
+Dynamic update of page content.
```

Comment:
I wouldn't say it adds views to SPAs, because if you can see something on the page, that's usually considered a View from an MVC perspective.
One other advantage to using Ember is that it takes care of AJAX calls for you.
.5/1